app/assets/javascripts/index.js.erb

```diff
@@ -243,6 +252,10 @@ class BirdMap {
 
 		const bounds = currentPosition.toBounds(m);
 
+		this.locations = L.markerClusterGroup();
+		this.location_markers = L.layerGroup([this.locations]);

```

Comment:
This should not be added; it already exists on line 263

Reply:
on line 255 and 256 I was trying to get the location markers to appear outside the clusterclick though I wasn't able to do so, is there a way to do this?

Reply:
Since `this.locations` already exists, I think you can just modify [line 264:](https://github.com/debbieyuen/baytrailbirding/blob/a4c232f5af3022d16a430220c8671e7eb6657b97/app/assets/javascripts/index.js.erb#L264)
```javascript
// From
this.locations = L.markerClusterGroup();

// To
this.locations = L.layerGroup();
```